Gemfile

```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+group :test do
+  gem 'rspec'
```

Comment:
I'm guessing both 2.14, 3.0, 3.1, 3.2 are compatible, and I don't really want to lock people into rspec 3 if the existing 2.14 works.

maybe just >= 2.14 ?